pystarport/cluster.py

```diff
@@ -1181,17 +1181,17 @@ def edit_tm_cfg(path, base_port, peers, config, *, custom_edit=None):
     # tendermint is start in process, not needed
     # doc['proxy_app'] = 'tcp://127.0.0.1:%d' % abci_port(base_port)
     rpc = doc["rpc"]
-    rpc["laddr"] = "tcp://0.0.0.0:%d" % ports.rpc_port(base_port)
-    rpc["pprof_laddr"] = rpc["pprof-laddr"] = "localhost:%d" % (
+    rpc["laddr"] = "tcp://127.0.0.1:%d" % ports.rpc_port(base_port)
+    rpc["pprof_laddr"] = rpc["pprof-laddr"] = "127.0.0.1:%d" % (
         ports.pprof_port(base_port),
     )
     rpc["timeout_broadcast_tx_commit"] = rpc["timeout-broadcast-tx-commit"] = "30s"
-    rpc["grpc_laddr"] = rpc["grpc-laddr"] = "tcp://0.0.0.0:%d" % (
+    rpc["grpc_laddr"] = rpc["grpc-laddr"] = "tcp://127.0.0.1:%d" % (
         ports.grpc_port_tx_only(base_port),
     )
     p2p = doc["p2p"]
     # p2p["use-legacy"] = True
-    p2p["laddr"] = "tcp://0.0.0.0:%d" % ports.p2p_port(base_port)
+    p2p["laddr"] = "tcp://127.0.0.1:%d" % ports.p2p_port(base_port)
```

Comment:
to avoid mix ipv4 and ipv6 when [LookupIP](https://github.com/cometbft/cometbft/blob/v0.34.29/p2p/netaddress.go#L97)